text/0003-multisig-minting/0003-multisig-minting.md

```diff
@@ -10,6 +10,7 @@
 ## Summary
 An advantage of splitting the coinstake transaction into a *monetary creation* and a *coin-age consumption* transaction, as described in [RFC-0002](../0002-split-coinstake-transaction/0002-split-coinstake-transaction.md), allows the *coin-age consumption* transaction to be pre-signed off-line.
 In combination with multi-signature scripts, this could serve as an alternative to *cold minting* by allowing for *air-gapped minting*.
+If a *cold minting* protocol is implemented, the conventions proposed could be used to implement secure socialization of pool rewards in a minting pool.
```

Comment:
Not sure secure is the right word here. It is rather socialisation without risk to lose your coins.
Secure could give the impression the reward output can be signed up front, which is not the case.